packages/pwa-kit-react-sdk/src/ssr/universal/extensibility/application-extension.ts

```diff
@@ -16,14 +16,14 @@ import {IRouteConfig} from './types'
  * @abstract
  */
 export default abstract class ApplicationExtension {
-    private config: Record<string, any>
+    private config: Record<string, unknown>
```

Comment:
Is there a reason why you didn't create a type specifically for the config value? I think this will help us in the future if we expand on our base extension configurations, beyond having enabled.

Something like this:

```
type IConfig = {
  [key: string]: any;  // Allows any string key with any value
  enabled?: boolean;   // Optional "enabled" property that must be a boolean
};
```